models/helpers.ts

```diff
@@ -190,6 +212,8 @@ export const statusColor: Record<
   NotSaved: 'gray',
   Submitted: 'green',
   Cancelled: 'red',
+  Return: 'orange',
+  ReturnIssued: 'gray',
```

Comment:
For now use these:
```
Return: 'green',
ReturnIssued: 'green',
```

Since both statuses don't imply that the user need any action they can be green. Later on I'll be splitting these into a different section separate from the doc status (draft, saved, submitted, cancelled).

Reply:
corrected